crates/routes/src/utils/setup_local_site.rs

```diff
@@ -79,9 +80,25 @@ pub async fn setup_local_site(pool: &mut DbPool<'_>, settings: &Settings) -> Lem
   };
   let site = Site::create(pool, &site_form).await?;
 
+  // create multi-comm follower account
+  let r: String = rand::rng()
```

Comment:
It'd probably be easier to take the first few chars of a uuid, which we already have as a dep.